spec/unit/classes/server_spec.rb

```diff
@@ -157,4 +157,78 @@ class { 'postgresql::globals':
       is_expected.to contain_class('postgresql::repo').with_version('99.5')
     end
   end
+
+  describe 'create config entries' do
+    let(:params) {{ :config_entries => {:config_entry => {:ensure => 'present', :name => 'port_spec', :value => '5432'}}}}
```

Comment:
To fix the tests, try changing the symbols (`:config_entry`, `:ensure` etc) to strings, or replace the whole value assigned to `:config_entries` to a string that's compatible with Puppet's DSL. rspec-puppet passes it through with little editing, and the Ruby symbol syntax isn't valid in a manifest.